utils.groovy

```diff
@@ -535,6 +535,11 @@ def AWSBuildUploadCredentialExists() {
 // Emits Slack message if set up, otherwise does nothing.
 def trySlackSend(params) {
     if (utils.credentialsExist([string(credentialsId: 'slack-api-token', variable: 'UNUSED')])) {
+        // Prefix all slack messages with the emoji representing the platform
+        // either being the fcos pipeline or the rhcos pipeline. Default to
+        // just a generic CoreOS emoji if none given.
+        def emoji = pipecfg.misc?.slack_platform_emoji_string ?: ':coreos:'
```

Comment:
The `pipecfg` isn't available here. We could pass it in though it's less elegant and messes up the nice parameter passthrough we do in this function. That's where the `jenkins-config` ConfigMap would be nicer since it's akin to "global" state and we could just requery it cheaply by calling `load_jenkins_config()`.

Short-term I guess let's just pass it as a keyed param and fetch it here via `params['pipecfg']`?

Reply:
ok pushed an update that will use an annotation on the secret instead